spooky-oysters/React-FrontEnd
# Gl setup details class

## Link to Ticket

no direct ticket

## Description of Proposed Changes

- added class to handle details view of training programs
- imported class into App.js to use in DOM

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout gl-setupDetailsClass
npm start
```

In Browser, navigate to:
http://bangazon.com:3000/

Steps to test:
Should see a single Training Program details such as Name, Start Date, End Date, Max Attendance


## Impacted Areas in Application

List general components of the application that this PR will affect:

* Details.js
* App.js

## Mentions @username

Tag users that need to review this code
